pilot.go

```diff
@@ -260,16 +260,35 @@ func initAutoPilot(svr *server, cfg *lncfg.AutoPilot,
 			chanState := make([]autopilot.LocalChannel,
 				len(activeChannels))
 			for i, channel := range activeChannels {
+				localCommit := channel.LocalCommitment
+				balance := localCommit.LocalBalance.ToSatoshis()
+
 				chanState[i] = autopilot.LocalChannel{
-					ChanID:   channel.ShortChanID(),
-					Capacity: channel.Capacity,
+					ChanID:  channel.ShortChanID(),
+					Balance: balance,
 					Node: autopilot.NewNodeID(
 						channel.IdentityPub),
 				}
 			}
 
 			return chanState, nil
 		},
+		ChannelInfo: func(chanPoint wire.OutPoint) (
+			autopilot.LocalChannel, error) {
+
+			channel, err := svr.remoteChanDB.FetchChannel(chanPoint)
+			if err != nil {
+				return autopilot.LocalChannel{}, err
+			}
+
+			localCommit := channel.LocalCommitment
+			return autopilot.LocalChannel{
+				ChanID:  channel.ShortChanID(),
+				Balance: localCommit.LocalBalance.ToSatoshis(),
+				Node: autopilot.NewNodeID(
```

Comment:
nit: no need for line break

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/4666/commits/92dcd47f267c1d2cd19523be37bb36d6bba4fc33